test/integration/resourcemanager/tokenrequestor/tokenrequestor_test.go

```diff
@@ -60,6 +63,8 @@ var _ = Describe("TokenRequestor tests", func() {
 				Namespace: testNamespace.Name,
 			},
 		}
+
+		fakeClock.SetTime(time.Now().Round(time.Second))
```

Comment:
Why did you introduce this fake clock in this PR?

Reply:
The reconciler panics if the clock is not set or the JitterFunc is not set, earlier this was recovered because we set `RecoverPanic` to true in the controller. Now we set it at manager level, but we don't do it in the tests.

Reply:
Cool, makes sense, thanks!